src/drt/src/io/io.cpp

```diff
@@ -855,6 +855,27 @@ void updatefrAccessPoint(odb::dbAccessPoint* db_ap,
       }
     }
   }
+  auto db_path_segs = db_ap->getSegments();
+  for(const auto& db_path_seg : db_path_segs) {
+    Rect db_rect = std::get<0>(db_path_seg);
+    bool begin_style_trunc = std::get<1>(db_path_seg);
+    bool end_style_trunc = std::get<2>(db_path_seg);
+
+    auto db_rect_points = db_rect.getPoints();
+    Point db_ul_point = db_rect_points[3];
+    Point db_lr_point = db_rect_points[1];
```

Comment:
Rect has methods for getting each point separately( ll(), ur(), ul(), lr())
The begin and end points are lower left and upper right. This should be updated.
@Mahfouz-z